string class_name
float64 confidence
int32 x_min
int32 x_max
int32 y_min
int32 y_max